mlflow/langchain/_langchain_autolog.py

```diff
@@ -34,10 +52,75 @@ def _patched_inference(original, self, *args, **kwargs):
 
     We patch inference functions for different models based on their usage.
     """
-    _MODEL_TRACKER.set_active_model_id_for_identity(id(self))
+    if model_id := _MODEL_TRACKER.get(id(self)):
+        _MODEL_TRACKER.set_active_model_id(model_id)
+    else:
+        _MODEL_TRACKER.set_active_model_id(None)
     return original(self, *args, **kwargs)
 
 
+def _patched_invoke(original, self, *args, **kwargs):
+    """
+    A patched implementation of langchain models invoke process which enables
+    logging the traces.
+    """
+    config = AutoLoggingConfig.init(flavor_name=mlflow.langchain.FLAVOR_NAME)
+    if model_id := _MODEL_TRACKER.get(id(self)):
+        _MODEL_TRACKER.set_active_model_id(model_id)
+    # NB: this check ensures we don't create LoggedModels for internal components
+    elif not _MODEL_TRACKER._is_active_model_id_set and config.create_logged_model:
+        logged_model = mlflow.create_logged_model(
+            name=self.__class__.__name__,
+        )
```

Comment:
Is it possible to set the status to READY?

<img width="796" alt="image" src="https://github.com/user-attachments/assets/805f111e-0cc6-4e84-b5c7-e2eff9e57db7" />